net/peer_tracker_test.go

```diff
@@ -159,6 +156,9 @@ func TestUpdateWithErrors(t *testing.T) {
 	err := tracker.UpdateTrusted(context.Background())
 	assert.Error(t, err, "all updates failed")
 
+	updatedHead := types.NewTipSetKey(types.CidFromString(t, "UPDATE"))
```

Comment:
This has nothing to do with the bug, just moving associated code closer together.